contrail_api_cli/commands/schema.py

```diff
@@ -0,0 +1,53 @@
+from ..command import Command, Arg, CommandError
```

Comment:
Add at the top

# -*- coding: utf-8 -*-
from __future__ import unicode_literals

To use unicode strings by default